chisel-wrapper

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# USAGE:
+#   ./<script-name> [ chisel-cut-OPTIONS ] <slices..>
+# Installs the slices, using
+#   chisel cut "$@"
+# Generates a status file similar to /var/lib/dpkg/status at the CHISEL_DPKG_STATUS_FILE path
+# env vars:
+#   CHISEL_BIN                (optional) the path of chisel binary
+#   CHISEL_CACHE_DIR          (optional) the path which chisel will use to store it's cache
+#   CHISEL_DPKG_STATUS_FILE   (required) the path where the dpkg status file should be generated
+
+set -eu
+
+CHISEL_BIN="${CHISEL_BIN:-"chisel"}"
+CHISEL_CACHE_DIR="${CHISEL_CACHE_DIR:-"$(mktemp -d)"}"
```

Comment:
I understand that the point of `mktemp -d` here is to have an empty cache directory on each invocation so that we can see what appears in the cache. But since we create a new unique temporary directory and fill it with stuff on every invocation, we should at least clean it afterwards. And if we're going to clean it on every run, it's probably of no use to the caller of this script to configure where it is (maybe aside from putting it on the same filesystem).

So I suggest:
```
CHISEL_CACHE_DIR=$(mktemp -d)
cleanup() {
    rm -rf "$CHISEL_CACHE_DIR"
}
trap cleanup EXIT
```

Reply:
You have a good point. Cris mentioned it before as well. I kept `CHISEL_CACHE_DIR` in case anyone wanted to use the existing cache directory, and didn't clean up because the directory might be used later on. But since these are use cases that we are not immediately using, I suppose I can just remove the variable entirely and cleanup as you suggested.

Reply:
> I kept CHISEL_CACHE_DIR in case anyone wanted to use the existing cache directory

But then the directory might exist and contain junk so you will have to clean it up first. Now you'd have two cases 1) a temporary directory that needs to be removed afterwards, 2) user provided directory that should be cleaned before. IMHO not worth the effort.

Reply:
Yeah, you are right. Added in https://github.com/canonical/rocks-toolbox/pull/1/commits/84640858c70ff3487b538ef482be1b48a21832aa. Thank you.